README.md

```diff
@@ -1,4 +1,4 @@
-# Skyux-Lib-Help
+# skyux-lib-help
 
 ### The SKYUX library for interacting with the Help Widget.
```

Comment:
`SKY UX` should be 2 words. ... I'd also change `for interacting` with `to interact`. ... And I don't think we need a period at the end here.